CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- **CUMULUS-3680**
+  - Fixed drop-down menus to include all collection and provider options .
+  - Removed unnecessary API calls from Operations and Providers pages.
+  - Fixed search on Operations overview page.
+- **CUMULUS-4094**
+  - Set the endDateTime to null for initial date range
```

Comment:
This is from internal slack and is a quick fix. "We'd like to request that 1) the end date not be populated so new information will not be filtered out."

Reply:
Are there any performance concerns here? Probably not but just asking as we're removing a constraint on an API query presumably.